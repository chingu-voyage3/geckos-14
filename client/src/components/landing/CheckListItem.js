import React from 'react';
const CheckListItem = props => {
  return (
    <div>
      <i className="fas fa-check-square fa-3x"></i> <h5>{props.text}</h5>
    </div>
  );
};

export default CheckListItem;
