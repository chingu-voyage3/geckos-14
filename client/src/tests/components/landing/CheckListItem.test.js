import React from 'react';
import CheckListItem from '../../../components/landing/CheckListItem';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<CheckListItem item='Discover all sensors/actuators connected to your device' />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
