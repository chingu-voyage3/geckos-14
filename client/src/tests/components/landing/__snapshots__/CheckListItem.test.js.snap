// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <i
    className="fas fa-check-square fa-3x"
  />
   
  <h5 />
</div>
`;
